File: model/truck.go
package model

type Truck struct {
	ID         string   `json:"id"`
	Name       string   `json:"name"`
	Twitname   string   `json:"twitname"`
	Weburl     string   `json:"weburl"`
	Retweeted  bool     `json:"retweeted"`
	Lastupdate int64    `json:"lastupdate"`
	Updated    string   `json:"updated"`
	Type       string   `json:"type"`
	About      string   `json:"about"`
	Foursquare string   `json:"foursquare"`
	Location   string   `json:"location"`
	Zone       string   `json:"zone"`
	Image      string   `json:"image"`
	Tweets     []*Tweet `json:"tweets"`
	Images     []*Image `json:"images"`
	Site       string   `json:"site"`
	Archive    bool     `json:"archive"`
}

File: src/Wanderinglunch.Data/Models/Truck.cs
namespace Wanderinglunch.Data.Models
{
    public class Truck
    {
        public string Name { get; set; }
        public string Id { get; set; }

        private string _webURL;

        public string WebURL
        {
            get => _webURL;
            set
            {
                if (value == null)
                {
                    _webURL = "";
                }
                else
                {
                    _webURL = value;
                }
            }
        }

        public long LastUpdate { get; set; }
        public string Type { get; set; }

        private string _about;

        public string About
        {
            get => _about;
            set
            {
                if (value == null)
                {
                    _about = "";
                }
                else
                {
                    _about = value;
                }
            }
        }

        private string _foursquare;

        public string Foursquare
        {
            get => _foursquare;
            set
            {
                if (value == null)
                {
                    _foursquare = "";
                }
                else
                {
                    _foursquare = value;
                }
            }
        }

        public string Site { get; set; }
        public bool Archive { get; set; }
    }
}